site/docs/guide/introduction.md

```diff
@@ -173,6 +173,34 @@ You can stop the bot again with `Ctrl+C`.
 Ready?
 [Get started](./getting-started#getting-started-on-deno)! :robot:
 
+### Prerequisites for Bun
+
+Bun is a new JavaScript runtime, which is another good choice to build bot, like Deno, you are going to write your bot in TypeScript.
+The exact commands for all of that will be introduced in the next section when you actually create a bot, but it is important to know that these steps are necessary.
+
+Firstly, you have to have [Bun](https://bun.sh/) installed.
+
+In summary, this is what you have to do for Bun:
+
+Create a new directory somewhere.
+It will contain your bot project.
+Open this new directory in VS Code.
+
+```sh
+mkdir ./my-bot
+cd ./my-bot
+code .
+```
+
+Then:
+
+1. Run `bun init --yes` in your terminal to initialize the project.
+2. Create a source file `bot.ts` with TypeScript code inside the project.
```

Comment:
Should change `bot.ts` to `index.ts` which bun created by default when using `bun init --yes`

Reply:
How can this be fixed?

Reply:
> How can this be fixed?

Just change filename

Reply:
Wait so there's no way to skip creating the file, or so create it with a different name?

Reply:
I don't guess. Bun by default create index.ts with content:

```ts
console.log("Hello via Bun!");
```

Reply:
I don't see how we need that. Note that we also don't run `deno init` for the same reason.

Reply:
> I don't see how we need that. Note that we also don't run `deno init` for the same reason.

This is to show that we are starting a clean project. Of course, we may not init the project and start a ready-made one.

Reply:
Since almost everything `bun init` does is wrong for our use case, I don't understand why it would be useful. The docs would be:
1. Run `bun init`
2. Delete all files it created
3. Create correct files

Reply:
It's not deleting already created files

Reply:
Read again